devenv.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+set -eo pipefail
+
+./webodm.sh checkenv
```

Comment:
This will only work if `$PWD` is the directory containing `devenv.sh`. An idiom to determine the directory containing the current file is:

```bash
__dirname=$(cd $(dirname "$0"); pwd -P)
```

So you can then reference `webodm.sh` as `${__dirname}/webodm.sh`